robyn/__init__.py

```diff
@@ -424,8 +424,9 @@ def configure_authentication(self, authentication_handler: AuthenticationHandler
 
 
 class SubRouter(Robyn):
-    def __init__(self, file_object: str, prefix: str = "", config: Config = Config()) -> None:
-        super().__init__(file_object, config)
+    def __init__(self, prefix: str = "", config: Config = Config()) -> None:
+        file_object = "/dev/null" if os.name != "nt" else "NUL"
```

Comment:
this is not the correct approach, unfortunately. Could you please discuss the approach to the issue first? 

This will help you save time during the implementation process.